tests/run-make/rustdoc-test-args/rmake.rs

```diff
@@ -0,0 +1,21 @@
+extern crate run_make_support;
+
+use run_make_support::{out_dir, rustdoc};
+use std::{fs, iter};
+use std::path::Path;
+
+fn generate_a_lot_of_cfgs(path: &Path) {
+    let content = iter::repeat("--cfg=a\n").take(75_000).collect::<String>();
+    fs::write(path, content.as_bytes()).expect("failed to create args file");
+}
+
+fn main() -> Result<(), ()> {
```

Comment:
and this shouldn't be necessary anymore.
```suggestion
fn main() {
```